public/data/abhayprajapati.json

```
@@ -0,0 +1,24 @@
+{
+    "name": "Abhay Prajapati",
+    "type": "personal",
+    "bio": "frontend dev",
+    "avatar": "https://github.com/theabhayprajapati.png",
+    "links": [
+      {
+        "name": "My Personal Page",
+        "url": "http://abhayprajapati.me",

```

Your link is using a secure protocol! 

```suggestion
        "url": "https://abhayprajapati.me",

```